tasks/setup-mysql.yml

```diff
@@ -52,7 +56,7 @@
 - name: setup semaphore database user
   mysql_user:
     name: "{{ semaphore_mysql_user }}"
-    host: "localhost"
+    host: "%"
```

Comment:
this is a security risk since we are setting a default password, for custom mysql settings the var `semaphore_mysql_install` can be set to `false`